### PR TITLE
290/390-p11-suggestion

### PR DESCRIPTION
**Pull request by ymzhang-neo**
Dear @kevinamstutz , I have made several suggestions in this pull request. For example, I capitalized the keywords for SQL queries just to stay consistent with Projects 9 and 10. I will add explanation to the changes next to the code.

In general, please feel free to accept or discard any of them. Thanks!

### CODE REVIEW

File: 08-projects.Rmd [hunk 1]
```diff
@@ -4815,53 +4815,74 @@ dbGetQuery(con, "SELECT * FROM battings LIMIT 5;")
...
 
 #### Dataset
 
-The following questions will use the `elections` database and the following database found in Scholar:
+The following questions will use the `elections` database. Similar to Project 10, this database is hosted on Scholar. Moreover, Question 1 also involves the following data files found in Scholar:
 
 `/class/datamine/data/election/itcontYYYY.txt` (for example, data for year 1980 would be `/class/datamine/data/electionitcont1980.txt`)
 
-A public sample of the data can be found [here](https://www.datadepot.rcac.purdue.edu/datamine/data/election/itcontYYYY.txt)
+A public sample of the data can be found [here](https://www.datadepot.rcac.purdue.edu/datamine/data/election/itcontYYYY.txt).
```
Comment (ymzhang-neo) [inline 08-projects.Rmd:4836]: @kevinamstutz , could you fix the link here?

File: 08-projects.Rmd [hunk 2]
```diff
@@ -4815,53 +4815,74 @@ dbGetQuery(con, "SELECT * FROM battings LIMIT 5;")
...
+con <- dbConnect(RMariaDB::MariaDB(), 
+                 host="scholar-db.rcac.purdue.edu", 
+                 db="elections", 
+                 user="elections_user", 
+                 password="Dataelect!98")
+```
+````
+When a question involves SQL queries in this project, you may use a SQL code chunk (with `{sql}`), or an R code chunk (with `{r}`) and functions like `dbGetQuery` as you did in Project 10. Please refer to Question 5 in the [project template](https://raw.githubusercontent.com/TheDataMine/the-examples-book/master/files/project_template.Rmd) for examples.
+
+##### 1. Approximately how large was the lahman database (use the sqlite database in Scholar: `/class/datamine/data/lahman/lahman.db`)? Use UNIX utilities you've learned about this semester to write a line of code to return the size of that .db file (in MB).
```
Comment (ymzhang-neo) [inline 08-projects.Rmd:4856]: I re-worded Question 1 to clarify the three tasks involved. However, I am not sure whether `##### text` would look identical to `**text**`...

File: 08-projects.Rmd [hunk 3]
```diff
@@ -4815,53 +4815,74 @@ dbGetQuery(con, "SELECT * FROM battings LIMIT 5;")
...
 
-##### 2. Write a SQL query using the `LIKE` command to find a unique list of `zip_code`s that start with "479". How many unique `zip_code`s are there that begin with "479"?
+##### 2. Write a SQL query using the `LIKE` command to find a unique list of `zip_code`s that start with "479". 
+
+**Write another SQL query and answer: How many unique `zip_code`s are there that begin with "479"?**
 
 **Hint:** Make sure you only select `zip_code`s.
 
-**Relevant topics:** [sql](#sql), like
+**Relevant topics:** [SQL](#sql), [LIKE](https://www.tutorialspoint.com/sql/sql-like-clause.htm)
```
Comment (ymzhang-neo) [inline 08-projects.Rmd:4885]: @kevinamstutz please feel free to change the link for `LIKE`.

File: 08-projects.Rmd [hunk 4]
```diff
@@ -4872,7 +4893,9 @@ The data in that folder has been added to the `elections` database in the `elect
 
 ##### 3. Write a SQL query that counts the number of donations (rows) that are from Indiana. How many donations are from Indiana? Rewrite the query and create an _alias_ for our field so it doesn't read `COUNT(*)` but rather `Indiana Donations`. 
 
-**Relevant topics:** [sql](#sql), [where](#sql-where), [aliasing](#sql-aliasing)
+**Hint:** You may enclose an alias name in quotation marks (single or double) when the name contains space.
```
Comment (ymzhang-neo) [inline 08-projects.Rmd:4896]: I added the hint simply from my experience: Previously I only have tried one-word alias names and thus not used quotation marks.

File: 08-projects.Rmd [hunk 5]
```diff
@@ -4893,17 +4916,19 @@ The data in that folder has been added to the `elections` database in the `elect
 
 **Hint:** Use CONCAT and aliasing to accomplish this.
 
-**Relevant topics:** [sql](#sql), [aliasing](#sql-aliasing), concat
+**Relevant topics:** [SQL](#sql), [aliasing](#sql-aliasing), [CONCAT](https://www.tutorialspoint.com/sql/sql-concat-function.htm)
```
Comment (ymzhang-neo) [inline 08-projects.Rmd:4919]: @kevinamstutz please feel free to use another reference/link for `CONCAT`.

File: 08-projects.Rmd [hunk 6]
```diff
@@ -5374,7 +5399,7 @@ The next couple of questions should be complete in the same language. You can us
 
 ##### 4. Now that you have a working query, connect to the database and run the query to get the data into an R or `pandas` data frame. In previous projects, we learned how to used regular expressions to search for text. For each character, how many episodes `primary_title`s contained their name? 
 
-**Relevant topics:** [SQL in R](#sql-in-r), [SQL in Python](#sql-in-python), [grep](#r-grep)
+**Relevant topics:** [SQL in R](#r-sql), [SQL in Python](#sql-in-python), [grep](#r-grep)
```
Comment (ymzhang-neo) [inline 08-projects.Rmd:5402]: Hi @kevinamstutz , while I changed the link, it seems that `#r-sql` is still an empty section in the Examples Book. Do you want to link the "SQL in R" section under the SQL chapter (https://thedatamine.github.io/the-examples-book/sql.html)?

Comment (kevinamstutz) [reply 08-projects.Rmd:5402]: I think I'm just going to go ahead and add a little bit of content to it so it's not still empty! Thanks!

**Comment by kevinamstutz** [comment]
Thanks @ymzhang-neo, this looks excellent!!

File: 08-projects.Rmd (hunk 6 above)
Comment (ymzhang-neo) [reply 08-projects.Rmd:5402]: Thanks for editing the book!

**Comment by ymzhang-neo** [comment]
Thank you @kevinamstutz for all the editing!

**Pull request merged**